packages/orchestrator-ui-components/src/components/WfoWorkflowSteps/WfoWorkflowStepList/WfoWorkflowStepList.tsx

```diff
@@ -17,18 +19,60 @@ export interface WfoWorkflowStepListProps {
     isTask: boolean;
 }
 
+interface SubscriptionDeltaState {
+    subscription: object;
+}
+
 export const WfoProcessRawData = ({ processId }: { processId: string }) => {
     const { data, isFetching } = useRawProcessDetails(processId);
     return isFetching ? <WfoLoading /> : <WfoJsonCodeBlock data={data || {}} />;
 };
 
+export const WfoProcessSubscriptionDelta = ({
+    processId,
+}: {
+    processId: string;
+}) => {
+    const { data, isFetching } = useRawProcessDetails(processId);
```

Comment:
Maybe a bit out of scope, but still related: why is this hook returning data of type 'object'? I suggest to apply a type inside the useRawProcessDetails hook, at least define a base type that the code in this component relies on. That would resolve the need for type casting below

Reply:
It returns a new type now